### DISCUSSION TITLE
Port for RStudio

### DISCUSSION THREAD

**Discussion by cwdjankoski** [daltonmenezes/aura-theme#116] [Port request]
Hello @daltonmenezes 
I tried out your theme for the terminal and it looks fantastic - thanks ! 
Would you please consider ports also for RStudio (perhaps helpful [info](https://rstudio.github.io/rstudio-extensions/rstudio-theme-creation.html)) and for Emacs ? 

Thanks.

**Comment by cwdjankoski** [comment]
Hi @daltonmenezes, 
Thank you for considering my issue. 

From the above link that I posted before for RStudio - I noticed that it is possible to import either `.tmTheme` files or custom RStudio ones `.rstheme`.  
So I tried to import the `aura-theme.tmTheme` file from the sublime-text subdir but unfortunately without success :-\  
The error mssg is:
```
Specification mandates value for attribute crossorigin[41]
```

I would like to try and help you further here even though I don't know what I am doing :-) Please feel free to tell me how could i help.

**Reply by daltonmenezes** [reply]
Hi @cwdjankoski , I think you can compare a `.tmTheme` from another theme that works on RStudio and see what are the differences from the Aura `.tmTheme`, I believe this is fastest way to produce this port. The common path is to get a `.rstheme` file and apply the Aura colors to it and finally produce the port specific to RStudio.

The [Color palettes](https://github.com/daltonmenezes/aura-theme/tree/main/packages/color-palettes) and [Documents](https://github.com/daltonmenezes/aura-theme#documentation) may will help you.
